languages/c/README.md

```diff
@@ -41,7 +41,7 @@ Before launch, we need all of the following parts to be completed:
 
 ### Test Runner
 
-- [ ] Build Test Runner
+- [x] [Build Test Runner] (https://github.com/exercism/c-test-runner)
```

Comment:
The link doesn't look right in the preview. I think it's because there is a space between the end bracket and beginning parentheses.

Reply:
D'oh!